utils/eza/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eza
+PKG_VERSION:=0.18.4
```

Comment:
My 2c: Hmm, hmm... they are releasing very often. I see there is a new version. This will be very hard to maintain in terms of keeping this up to date. One guy complained recently that so many packages are so old in this repo... Dont take it wrong. Just my thoughts.

Reply:
None taken.
Concern about this is in place. I cannot volunteer to update it once in a week if they keep up with that pace.